.github/workflows/e2etest-run-modulestest.yml

```diff
@@ -59,41 +59,31 @@ jobs:
       - name: Run modulestest
         if: success()
         run: |
-          cd build
+          recipes=$(ls -d ../src/modules/test/recipes/*.json | grep -v SampleTests.json)
+          bin="./modules/bin/"
+          runNumber=1
+
           function checkResult(){
             if [[ $1 -ne 0 ]]; then
               echo '::error title=E2E Test Failure::The E2E tests failed at run number $runNumber'
               exit $1
             fi
           }
-          runNumber=1
+
+          cd build
+
           if [[ ${{ inputs.repeatForXMins }} -gt 0 ]];then
             end=`date -d "+${{inputs.repeatForXMins}} mins" +%s`
             while [[ $(date +%s) -lt $end ]]; do
               echo Performing test run $((++runNumber))
-              ./modulestest --gtest_output=xml:gtest-output/TestRecipes.xml
```

Comment:
is this no longer needed? No more gtest integration?

Reply:
Yes correct